### ISSUE TITLE
Incomplete RTO Go README file instruction

### ISSUE THREAD

**Issue by wasin-waeosri** [thomsonreuters/websocket-api#22]
Hi Team,

I have followed the steps in /RDP/go/README.md, but I cannot run the market_price_rdpgw_client_cred_auth.go example. It always returns "no required module provides package github.com/gorilla/websocket" message.

go version go1.17.12 linux/amd64

Example Logs:
```
[root@ddef5e35bfd4 go]# go get github.com/gorilla/websocket
go: downloading github.com/gorilla/websocket v1.5.0

[root@ddef5e35bfd4 go]# go run market_price_rdpgw_client_cred_auth.go --clientid $RTO_CLIENTID_V2_1 --clientsecret $RTO_CLIENTSECRET_1

market_price_rdpgw_client_cred_auth.go:28:2: no required module provides package github.com/gorilla/websocket: go.mod file not found in current directory or any parent directory; see 'go help modules'
```
According to my research, A go app first needs to be initialized with ```go mod init``` before we can use external modules. I tested it and the example works fine.

Example:
```
[root@ddef5e35bfd4 go]# go mod init myapp
go: creating new go.mod: module myapp
go: to add module requirements and sums:
        go mod tidy
[root@ddef5e35bfd4 go]# go get github.com/gorilla/websocket
go get: added github.com/gorilla/websocket v1.5.0

[root@ddef5e35bfd4 go]# go run market_price_rdpgw_client_cred_auth.go --clientid $RTO_CLIENTID_V2_1 --clientsecret $RTO_CLIENTSECRET_1
2022-09-12 11:16:32.262737 Sending authentication request...
2022-09-12 11:16:33.935777 RECEIVED:
{
  "access_token": "<access_token>",
  "expires_in": 7199,
  "token_type": "Bearer"
}
..
``` 

Best regards,

Wasin W.

**Comment by L-Karchevska** [comment]
@wasin-waeosri Thank you for pointing up the gap in README and the research effort, we will ix this in the upcoming releases.

**Comment by vlevendel** [comment]
@wasin-waeosri  Addressed with tag, WSA-1.1.11.
